composer.json

```diff
@@ -19,6 +19,7 @@
         "drupal/core-composer-scaffold": "^10.3.6",
         "drupal/core-dev": "^10.3.6",
         "drupal/core-recommended": "^10.3.6",
+        "drupal/groupmedia": "^4",
```

Comment:
I have not really evaluated the groupmedia module.
I would say keep it in this first version of the epic branch, and evaluate it further as we add more features.
Worst case is we remove it later.
Unless you want me to test something specific @AaronGilMartinez

Reply:
Indirect dependencies are fine in this case, as it's a mandatory dependency. Groupmedia works with group, it's a mandatory dependency. It would be different in case of a library.

Reply:
Changed

Reply:
Ok